R/run.R

```diff
@@ -396,26 +412,28 @@ CmdStanProcs <- R6::R6Class(
       private$total_time_ <- as.numeric(time)
       invisible(self)
     },
-    all_finished = function() {
-      finished <- TRUE
-      for (id in self$run_ids()) {
-        # if chain is not finished yet
+    check_finished = function() {
+      new_failed_procs = NULL
+      new_successful_procs = NULL
```

Comment:
I don't see `new_failed_procs` or `new_successful_procs ` used anywhere in `check_finished()`. They're returned at the end of the function but won't they always be `NULL`?

Reply:
This is leftover from a previous implementation (I redid this a couple of times before settling on this). Will cleanup.